.github/workflows/cron.yml

```diff
@@ -0,0 +1,35 @@
+name: Update BTFHub Archives
+on:
+  schedule:
+  - cron: '0 8 * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Update BTF
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Check out code repo
+      uses: actions/checkout@v2
+
+    - name: Checkout public BTFHub repo
+      uses: actions/checkout@v2
+      with:
+        repository: aquasecurity/btfhub
+        token: ${{ secrets.REPO_ACCESS_TOKEN }}
```

Comment:
Note: We'll need to generate this token for the organization and have it stored in the btfhub-tools repo for this action to be able to clone and commit to the btfhub repo.